### ISSUE TITLE
When clicking on dashboard items, first item of kill ring is yanked into the buffer

### ISSUE THREAD

**Issue by japanoise** [emacs-dashboard/emacs-dashboard#45]
1. Install dashboard (snippet from [my init.el](https://github.com/japanoise/dotemacs/blob/master/init.el) below)
2. Click on recent file
3. Text is yanked into the buffer.

```elisp
(use-package dashboard
  :ensure t
  :diminish dashboard-mode
  :config
  (setq dashboard-banner-logo-title "EmacsOS - Ready.")
  (setq dashboard-startup-banner 'official)
  (setq dashboard-items '((recents  . 10)
                          (projects . 10)
                          (agenda . 5)))
(dashboard-setup-startup-hook))
```

**Comment by japanoise** [comment]
So after some investigation, I discovered using `C-h k` that for some reason, mouse2 is the up event when clicking on these widgets with mouse1. Workaround that stops the defect, but doesn't 'fix' the bug as it's just hacking around the problem:
```elisp
(use-package dashboard
  :bind (
	 :map dashboard-mode-map
	 ("<down-mouse-1>" . nil)
	 ("<mouse-1>" . widget-button-click)
	 ("<mouse-2>" . widget-button-click))
  :config
  (setq dashboard-startup-banner 'official)
  (setq dashboard-items '((recents  . 10)
                          (projects . 10)
                          (agenda . 5)))
  (dashboard-setup-startup-hook))
```

**Comment by rakanalh** [comment]
Hi @japanoise 
I've seen this happening before when emacs starts with errors. But as far as i am concerned, dashboard doesn't define any mouse-specific functionality. I'll try to look into this soon, i suspect this is an issue with `recents` module.
Could you please try to confirm that this only happens when clicking recent files?

**Comment by japanoise** [comment]
Happens for all menu items as far as I can see.

**Comment by rakanalh** [comment]
I believe this should be fixed after having merged #91

**Comment by japanoise** [comment]
Can confirm that the issue no longer occurs after updating dashboard and with the hack mentioned in my earlier comment removed. Thank you :smile:
